grafana-cloud-metrics.yaml

```diff
@@ -118,14 +118,14 @@ outputs:
 
 resources:
   wait_condition:
-    type: "OS::Heat::SwiftSignal"
+    type: "AWS::CloudFormation::WaitCondition"
```

Comment:
These are actually the same resource under the hood. AWS::CloudFormation::WaitCondition is an alias for compatibility purposes.